packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -34,8 +33,8 @@ export interface CreateOpts {
 }
 
 interface HttpLog {
-  request: object;
-  response?: object;
+  request: Record<string, unknown>;
```

Comment:
Could use the Request/Response types from Express maybe if it's matching?

Reply:
nope, those are different objects